index.js

```diff
@@ -0,0 +1,136 @@
+import BookManager from './modules/bookManager.js';
+
+const bookManager = new BookManager();
+const listBooks = document.querySelector('#book_list');
+
+const deleteItem = (container, id) => {
+  listBooks.removeChild(container);
+  bookManager.deleteBook(id);
+};
+
+const createItem = (title, author, id) => {
+  const divContainer = document.createElement('div');
+  const titleAuthorElement = document.createElement('p');
+  const deleteButton = document.createElement('button');
+
+  divContainer.id = `book_${id}`;
+  divContainer.classList.add('book-details');
+
+  titleAuthorElement.innerText = `${title} by "${author}`;
+  titleAuthorElement.classList.add('text-author');
+  titleAuthorElement.classList.add('text-title');
+
+  deleteButton.innerText = 'Remove';
+  deleteButton.classList.add('delete-button');
+
+  deleteButton.addEventListener('click', () => {
+    deleteItem(divContainer, id);
+  });
+
+  divContainer.appendChild(titleAuthorElement);
+  divContainer.appendChild(deleteButton);
+  return divContainer;
+};
+
+const addItem = (book) => {
+  listBooks.appendChild(createItem(book.title, book.author, book.id));
+};
+
+const createBookList = () => {
+  bookManager.books.forEach((book) => {
+    addItem(book);
+  });
+};
+
+const addForm = document.getElementById('books__add');
+const titleForm = document.getElementById('title');
+const authorForm = document.getElementById('author');
+
+const addingBook = (e) => {
+  e.preventDefault();
+  const title = titleForm.value;
+  const author = authorForm.value;
+  bookManager.createBook(title, author);
+  addItem(bookManager.books[bookManager.books.length - 1]);
+  titleForm.value = '';
+  authorForm.value = '';
+};
+
+const addButtonListener = () => {
+  addForm.addEventListener('submit', addingBook);
+};
+
+const updateTime = () => {
+  const showTime = document.getElementById('date');
+  const date = new Date();
+  const year = date.getFullYear();
+  const month = date.toLocaleString('en', { month: 'long' });
+  const day = date.getDate();
+  const hour = date.getHours();
+  const minute = date.getMinutes();
+  const second = date.getSeconds();
+  const time = `${month} ${day} ${year}, ${hour}:${minute}:${second}`;
+  showTime.innerText = time;
+};
+
+const startTime = () => {
+  updateTime();
+  setInterval(updateTime, 1000);
+};
```

Comment:
- [ ] According to the project requirements, you are required to use the [Luxon](https://moment.github.io/luxon/#/install?id=es6) library for your time and data and not a custom function for this section. Kindly include the Luxon library and use it to generate your date and time details.

![image](https://user-images.githubusercontent.com/86318284/183965978-efbf90b5-5f69-4a61-bb34-2c3a943db094.png)